tools/testbench/file.c

```diff
@@ -459,16 +459,18 @@ static struct comp_dev *file_new(const struct comp_driver *drv,
 	case FILE_READ:
 		cd->fs.rfh = fopen(cd->fs.fn, "r");
 		if (!cd->fs.rfh) {
-			fprintf(stderr, "error: opening file %s\n", cd->fs.fn);
+			fprintf(stderr, "error: opening file %s for reading - %s\n",
+				cd->fs.fn, strerror(errno));
 			free(cd);
 			free(dev);
 			return NULL;
 		}
 		break;
 	case FILE_WRITE:
-		cd->fs.wfh = fopen(cd->fs.fn, "w");
+		cd->fs.wfh = fopen(cd->fs.fn, "w+");
```

Comment:
What's the purpose of w+?

Reply:
I was having issues opening this file, but this could have been related to the other issue below.
```
     w      Truncate file to zero length or create text file for writing.  The stream is positioned at the beginning of the file.

       w+     Open for reading and writing.  The file is created if it does not exist, otherwise it is truncated.  The  stream  is  posi‐
              tioned at the beginning of the file
```